packages/vuetify/src/locale/en.ts

```diff
@@ -61,4 +61,9 @@ export default {
       currentPage: 'Current Page, Page {0}',
     },
   },
+  rating: {
+    ariaLabel: {
+      icon: 'Rating Star {0}',
```

Comment:
Not sure if `Star` needs to be part of the text. The actual icon could be anything, not necessarily stars. I also think it would be useful to include the max number, something like `Rating x of y`